pkg/graveler/graveler.go

```diff
@@ -1196,7 +1463,28 @@ func (g *KVGraveler) DumpCommits(ctx context.Context, repositoryID RepositoryID)
 }
 
 func (g *KVGraveler) DumpBranches(ctx context.Context, repositoryID RepositoryID) (*MetaRangeID, error) {
-	return g.db.DumpBranches(ctx, repositoryID)
+	repo, err := g.GetRepository(ctx, repositoryID)
+	if err != nil {
+		return nil, err
+	}
+	iter, err := g.RefManager.ListBranches(ctx, repositoryID)
+	if err != nil {
+		return nil, err
+	}
+	defer iter.Close()
+	schema, err := serializeSchemaDefinition(&BranchData{})
+	if err != nil {
+		return nil, err
+	}
+	// TODO (niro): Replace by KV committedManager
```

Comment:
Same here

Reply:
It's just a TODO for me to validate the current flow as all parts come together